fedora/infra.py

```diff
@@ -101,7 +101,10 @@ async def oncall_add(self, evt: MessageEvent, username: str) -> None:
             # Just grab the first mxid in the users list on fas
             mxid = mxids[0]
         try:
-            await self.plugin.database.execute(dbq, fasusername, mxid, user.get("timezone", "UTC"))
+            timezone_to_insert = user.get("timezone")
+            if timezone_to_insert is None:
+                timezone_to_insert = "UTC"
+            await self.plugin.database.execute(dbq, fasusername, mxid, timezone_to_insert)
```

Comment:
Another, somewhat silly way to handle this is to have
```suggestion
            await self.plugin.database.execute(dbq, fasusername, mxid, user.get("timezone", "UTC") or "UTC")
```

The `default` argument to `get` handles the case where the key doesn't exist in the dictionary, and the `or "UTC"` handles the case where it _does_ exist, but isn't "truth-y", like if it's `None`. I'm not sure myself that it's a clearer approach, so pick whichever one you like.